Dockerfile

```diff
@@ -1,25 +1,22 @@
-FROM alpine:3.4
+FROM frolvlad/alpine-glibc
 MAINTAINER Aurélien Thieriot <aurelien@scalar.is>
 
-ENV METRICBEAT_VERSION=5.0.0-alpha4
+ENV METRICBEAT_VERSION=5.0.0-beta1
 
-RUN apk update && \
-    apk add \
+RUN apk add --no-cache \
       ca-certificates \
-      curl && \
-    rm -rf /var/cache/apk/*
```

Comment:
no because we use `--no-cache` see https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache